packages/flutter/lib/src/material/material.dart

```diff
@@ -91,10 +91,11 @@ abstract class MaterialInkController {
 /// of material cannot be printed beyond the bounds of the material.
 ///
 /// If the layout changes (e.g. because there's a list on the paper, and it's
```

Comment:
paper?

Reply:
heh, i still think of it as "quantum paper"...

Reply:
fixed -> material